Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -304,10 +312,20 @@ def setup(self):
     self.videoExportCheckBox.connect('toggled(bool)', self.videoFormatWidget, 'setEnabled(bool)')
     self.videoFormatWidget.connect("currentIndexChanged(int)", self.updateVideoFormat)
     self.singleStepButton.connect('toggled(bool)', self.numberOfStepsSliderWidget, 'setDisabled(bool)')
+    self.maxFramesWidget.connect('textChanged(QString)', self.maxFramesChanged)
 
     self.updateVideoFormat(0)
     self.updateViewOptions()
 
+  def maxFramesChanged(self):
+    maxFramesText = self.maxFramesWidget.text
+    # An exception is raised if maxFramesText is empty. Catch it and set it to 1 instead.
+    try:
+      maxFrames = int(maxFramesText)
+    except Exception as e:
```

Comment:
Generally speaking, it is recommended to catch more specific exception (in that case, that would be `ValueError`). That said, by using a spinbox, you wouldn't have to worry about this

Reply:
OK, I redid it with a QSpinBox. For some reason I thought using a QLineEdit would be easier, but you're absolutely right - the QSpinBox was simpler and shorter.

What's the best way to proceed - submit a new pull request?